uaclient/security.py

```diff
@@ -1039,15 +1040,25 @@ def _check_subscription_is_expired(cfg: UAConfig) -> bool:
 
     :returns: True if subscription is expired and not renewed.
     """
-    from uaclient import cli
+    effective_to_date = (
```

Comment:
This doesn't feel like it belongs in security.py. Can we bake this logic into a propert decorated method on UAConfig of some sort. Ultimately we'll need something like days_until_expiry in order to allow us to emit various MOTD/apt messages based on contracts expiring soon, contracts only expired less than a grace period (14 days) and contracts that have expired longer than the grace period. Also it'd be good to keep all json extraction logic for this effectiveTo key inside UAConfig so other callsites don't have to replicate error handling (and because eventually we'll have to support multiple contracts with potentially different expiry dates which will be surfaced under something like the contract ["items" list](https://github.com/CanonicalLtd/ua-contracts/blob/master/docs/contracts.yaml#L438) which you can see in current contract responses (and currently contains a single item list with the same effectiveTo key.